src_plugins/enum/dune

```diff
@@ -9,6 +8,6 @@
  (public_name ppx_deriving.enum)
  (synopsis "[@@deriving enum]")
  (preprocess
-  (action (run ppxfind -legacy ppx_tools.metaquot --as-pp %{input-file})))
- (libraries compiler-libs.common ppx_tools ppx_deriving.api)
+  (pps ppxlib.metaquot))
+ (libraries compiler-libs.common ppxlib ppx_deriving.api)
```

Comment:
Plugin authors will have to change their build system in a similar way to this. Releases of the plugin with this new style will not be compatible with older versions of ppx_deriving -- but they are compatible with older OCaml releases. (@thierry-martinez explains: the reason why we need a change is that ppxlib's metaquot is fixed on 4.08, while ppx_tools.metaquot works on the current OCaml version.)

Reply:
So plugin authors will have to add an upper bound on their older releases to the latest non-ppxlib version, and a lower bound on their new ppxlib-friendly releases on the new ppx_deriving version. But both kind of versions will support existing OCaml releases, and the code changes are very small in simple cases.

Reply:
Is `compiler-libs.common` still needed as a dependency here btw? Shouldn't we be using `ocaml-compiler-libs`?